apps/src/templates/instructions/Tab.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {PropTypes} from 'react';
+import color from "../../util/color";
+
+var styles = {
+  tab: {
+    marginRight: 5,
+    paddingLeft: 10,
+    paddingRight: 10,
+    paddingBottom: 6,
+    fontWeight: "bold",
+    color: color.charcoal,
+    cursor: "pointer",
+  }
+};
+
+const Tab = React.createClass({
```

Comment:
For all of our new React, we're trying to use `class` instead of `createClass`

i.e. `class Tab extends Component {`

(you will then need to import Component from React)

Reply:
I might also call this something like InstructionsTab, just to disambiguate it from other Tab components we may create in the future.